### PR TITLE
fix: Add type check for conversation ID

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/conversation/ConversationRepository.js [hunk 1]
```diff
@@ -547,11 +552,7 @@ z.conversation.ConversationRepository = class ConversationRepository {
       })
       .catch(error => {
         if (error.type !== z.conversation.ConversationError.TYPE.NOT_FOUND) {
-          this.logger.log(
-            this.logger.levels.ERROR,
-            `Failed to get conversation '${conversation_id}': ${error.message}`,
-            error
-          );
+          this.logger.error(`Failed to get conversation '${conversation_id}': ${error.message}`, error);
```
Comment (ffflorian) [inline app/script/conversation/ConversationRepository.js:555]: `this.logger.levels.ERROR` is not needed anymore?

Comment (gregor) [reply app/script/conversation/ConversationRepository.js:555]: No. This `this.logger.log(this.logger.levels,ERROR)` is the same as `this.logger.error()`.

Comment (lipis) [reply app/script/conversation/ConversationRepository.js:555]: Same for `info`, `warning`?

Comment (gregor) [reply app/script/conversation/ConversationRepository.js:555]: Si senor - I changed most of them a long time ago.

**Pull request merged**